src/components/Insight/InsightCardSmall.module.scss

```diff
@@ -1,12 +1,18 @@
 .title {
-  display: inline-block;
   font-size: 16px;
   line-height: 24px;
   color: var(--mirage);
   max-width: 100%;
-  overflow-x: hidden;
+  margin-bottom: 6px;
+  overflow: hidden;
   text-overflow: ellipsis;
-  white-space: nowrap;
+  word-wrap: break-word;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```

Comment:
Custom and cross-platform component for multiline text truncating -> [`MultilineText`](https://github.com/santiment/app/blob/master/src/components/MultilineText/MultilineText.js)

Reply:
done